cfme/storage/volume.py

```diff
@@ -294,8 +306,13 @@ def create_snapshot(self, name, cancel=False, reset=False):
                 view.save.click()
                 return snapshot_collection.instantiate(name, self.provider)
 
-    def attach_instance(self, name, mountpoint=None, cancel=False, reset=False):
-        view = navigate_to(self, 'AttachInstance')
+    def attach_instance(self, name, mountpoint=None, cancel=False, reset=False, storage_manager=None):
```

Comment:
@digitronik First problem is here. I can't use self.obj.name in View so I have to get entity here and do double navigation.